src/Helpers.js

```diff
@@ -0,0 +1,18 @@
+// no cyrillic letters allowed
+function latinCheck(string) {
+  const cyrillic = /.*[а-яА-ЯёЁ]/g;
+  return string.search(cyrillic) === -1;
```

Comment:
Добавляйте плз пустые строки перед return
```suggestion

  return string.search(cyrillic) === -1;
```